test/e2e/run_test.go

```diff
@@ -455,8 +455,8 @@ var _ = Describe("Podman run", func() {
 	})
 
 	It("podman run powercap is masked", func() {
-		if os.Getenv("CI") != "" {
-			Skip("CI VMs do not have access to powercap")
+		if _, err := os.Stat("/sys/devices/virtual/powercap"); err != nil {
```

Comment:
non blocking, but I guess we like to use fileutils.Exists() now?

Reply:
I haven't converted the tests, but you are right, let's use it!

Fixed and pushed a new version.

Reply:
I don't see a new push